src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,51 @@
+package duke;
```

Comment:
Small issue here but I thought it would be more apt to place the TaskList.java in your Task package where all the Task related classes are.